src/rpc/blockchain.cpp

```diff
@@ -438,7 +438,9 @@ static UniValue getdifficulty(const JSONRPCRequest& request)
 
 static std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,                 (numeric) transaction size in bytes\n"
+    return "    \"vsize\" : n,                (numeric) virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted.\n"
```

Comment:
15637: bip141 is segwit which we don't have, need to rephrase this, smth like "virtual transaction size. This can be different from actual serialized size for high-sigop transactions."

Reply:
done